src/app.css

```diff
@@ -0,0 +1,46 @@
+* {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-weight: inherit;
+	font-style: inherit;
+	font-size: 100%;
+	font-family: inherit;
+	vertical-align: baseline;
+	box-sizing: border-box;
+}
+
+:root {
+	--color-primary: #99c1b9;
+	--color-link: #8e7dbe;
+  --color-error: #d88c9a;
+	--color-font-primary: #000000;
+	--color-font-secondary: #909090;
+	--color-background-primary: rgba(0, 0, 0, 0.8);
+	--color-background-secondary: #f5f5f5;
+
+	--color-white: #ffffff;
+	--color-black: #000000;
+	--color-grey: #909090;
+
+	font-family: 'Inter', sans-serif;
+}
+
+@supports (font-variation-settings: normal) {
+  :root { font-family: 'Inter var', sans-serif; }
+}
+
+body {
+	height: 100vh;
```

Comment:
There is no need to define the height for the body, use it only on pages where it is needed.